src/main/java/org/apache/sysds/lops/Unary.java

```diff
@@ -122,7 +122,12 @@ private String getOpcode() {
 	}
 	
 	public static boolean isMultiThreadedOp(OpOp1 op) {
-		return op==OpOp1.CUMSUM
+		if(op == OpOp1.INVERSE || op == OpOp1.CHOLESKY || op == OpOp1.DETECTSCHEMA)
```

Comment:
What is the reason to change which ops are multi-threaded?

Reply:
All Unary Ops are multithreaded (except Cholesky and Inverse). Therefore, we had to add the other operators so that they can be compiled with the in-place flag.